test/dapp/index.html

```diff
@@ -6,22 +6,9 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
I kept this empty page voluntarily, so as not to load any remote page, and have to wait for any network requests. I really think it's cleaner than loading any website. Let me know if you have a strong opinion regarding this.